mmv1/products/networkservices/terraform.yaml

```diff
@@ -45,6 +45,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           resource_name_2: "my-fallback"
         ignore_read_extra:
          - "timeout"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "network_services_edge_cache_origin_v4auth"
+        primary_resource_id: "default"
+        vars:
+          resource_name: "my-origin"
+          access_key_id: "ACCESSKEYID"
```

Comment:
nit: We likely want to bake this value into the config, `vars` is mostly for resource names, which need to be unique.

Reply:
Thanks for pointing that out - removed the `access_key_id` var, but added `secret_name` which could cause issues if not unique.